.nuspec/Microsoft.Maui.Controls.DefaultItems.props

```diff
@@ -18,5 +18,5 @@
     <MauiCss Include="**\*.css" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder);$(DefaultWebContentItemExcludes)" />
   </ItemGroup>
 
-  <Import Project="Microsoft.Maui.Controls.SingleProject.AutoImport.props" />
+  <Import Project="AutoImport.InTree.props" />
```

Comment:
Renaming this file to InTree to match the rest of the files that are "swapped out" with workload files.